### PR TITLE
lakeFS 1.3.0 changelog

### PR DESCRIPTION
**Pull request by idanovo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+# v1.3.0
+
+:new: What's new:
+
+- Update aws go sdk v2 to support s3 express buckets (#7083)
+- Support long-running dump and restore (#6975)
+- lakectl: Improved performance for lakectl rm recursive command (#7035)
+- PythonSDK: Support StoredObject 'open' as both context and function(#7038)
```
Comment (nopcoder) [inline CHANGELOG.md:10]: remove

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
 # Changelog
 
+# v1.3.0
+
+:new: What's new:
+
+- Update aws go sdk v2 to support s3 express buckets (#7083)
```
Comment (nopcoder) [inline CHANGELOG.md:7]:
```suggestion
- Update AWS Go SDK v2 to support S3 express directory buckets (#7083)
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+- Support long-running dump and restore (#6975)
+- lakectl: Improved performance for lakectl rm recursive command (#7035)
```
Comment (nopcoder) [inline CHANGELOG.md:9]:
```suggestion
- lakectl: Improved performance for fs rm recursive command (#7035)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+- PythonSDK: Support StoredObject 'open' as both context and function(#7038)
+- Glue Exporter: Support Hadoop directory markers (#7058)
+
+
+:bug: Bugs fixed:
+
+- API: Fix StatsObject returned metadata on empty metadata (#7026)
+- Return the right default pagination default per page- 100 instead of 1000 (#7051)
```
Comment (nopcoder) [inline CHANGELOG.md]: API?

**Review by nopcoder — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (idanovo) [reply CHANGELOG.md:10]: wasn't sure because it had the include-changelog tag

File: CHANGELOG.md (hunk 4 above)
Comment (idanovo) [reply CHANGELOG.md]: yes

**Pull request merged**